docs/alerts/README.md

```diff
@@ -0,0 +1,30 @@
+# Prometheus rules GitOps
+
+Any Prometheus rules file defined in the
+[fleet/lib/prometheus-alertrules/rules](../../prometheus-alertrules/rules)
+directory will be deployed to the cluster. It's possible to define a default
+namespace in the `values.yaml` file with the `rules.namespace` key.
+
+## Adding Prometheus rules
+
+1. Write the Prometheus rules in a yaml file according to the [prometheus
+   specification](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/).
+1. Add the YAML file to the `/rules` directory
+1. Commit
+
+## Prometheus rule AURA standards
+
+* `summary` annotation: The `summary` annotation is used to be able to describe a
+  group of alerts incomming. This annotation DOES NOT contain any templated
```

Comment:
I've written this quite globally for all kinds of levels of Prometheus alerting understanding. The tricky part is that you want the summary to be more or less the same for all the alerts in the same grouping. The risk of simply allowing templated variables in the summary is that each summary will be unique, making alert grouping less useful IMHO.

Not really a technical limitation, more dependent on the "alert standard".

Reply:
I am having trouble coming up with a scenario where the summary would need to be used for grouping instead of the alertname.  I think that having interpolated values in the summary is useful information.  Alert manager's grouping is rather primitive and in most cases, we don't want any grouping unless there is a high alert volume, which alertmanager isn't capable of doing.  I suspect we will end up shipping everything to squadcast ungrouped anyways.

Reply:
I'll adjust the config and standard to that use case then (no grouping unless sepcified). In that use case the summary can really be templated as wanted.

In what I described, the summary wouldn't really be used to group. Say you group on `alertname,cluster`. The summary could then be used to provide short information bursed about the alert, without overloading the alert message with specific details if 25 pods are crashing in a single cluster. The only "useful" template value then used on the summary would be `cluster` really, so it's sometimes easier to avoid them completely.

However if grouping is the exception, rather then the norm (as would be the case if alerts are send to subsequent systems), the summary can be formatted however. Although in that case I'd argue that the difference between a `summary`/`description` is neglectable and you'd might as well just use the description.